hack/chains/check-taskrun.sh

```diff
@@ -114,29 +120,17 @@ say $SIGNATURE
 pause
 
 title Taskrun payload
-[[ -z $QUIET ]] && echo "$PAYLOAD" | yq-pretty
+[[ -z $QUIET ]] && echo "$RAW_PAYLOAD" | yq-pretty
 
 pause
 
-# Keep going if the verify fails
-set +e
-
-##
-## Fixme: This only works with artifacts.taskrun.format set to 'tekton'.
-## I've never been able to use cosign verify-blob to verify a task's
-## payload and signature using the 'in-toto' format and I don't know why.
-##
 # Now use cosign to verify the signed payload
 title Taskrun verification
 show-then-run cosign verify-blob --key $SIG_KEY --signature $SIG_FILE $PAYLOAD_FILE
-COSIGN_EXIT_CODE=$?
 
 # For debugging...
 title "To view taskrun"
 say " env EDITOR=view kubectl edit $TASKRUN_NAME"
 
 # Clean up
 rm $SIG_FILE $PAYLOAD_FILE
-
-# Use the exit code from cosign
-exit $COSIGN_EXIT_CODE
```

Comment:
The reason for the set +e and exit code hoop jumping was to ensure the `rm $SIG_FILE ..` clean up happened if the verify failed.

Probably not that important, but I remember now why I did that...

Reply:
Ah ok. In that case, maybe `trap` right after the files are created is a good idea:
```bash
# Cosign needs files on disk to do a verify-blob afaict
SIG_FILE=$( mktemp )
PAYLOAD_FILE=$( mktemp )
trap "rm -f ${SIG_FILE} ${PAYLOAD_FILE}" EXIT
echo -n "$PAYLOAD" > $PAYLOAD_FILE
echo -n "$SIGNATURE" > $SIG_FILE
```

Reply:
Yeah that's sensible, but imo it doesn't particularly need to be in this PR.

Reply:
I've added a [trap](https://github.com/redhat-appstudio/infra-deployments/pull/183/files#diff-0aaca8058d25ea8b1a3abd31c7d3eb390726d1ab896ef5829ad0b67b5e187390R114-R117) to cleanup the temp files.